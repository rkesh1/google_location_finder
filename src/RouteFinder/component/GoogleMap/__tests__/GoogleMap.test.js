import React from 'react';
import { shallow } from 'enzyme';
import GoogleMapRoute from '../index';


describe('GoogleMap Component Test Case ', () => {
  let wrapper;
  const props = {
    map:{},
    cordinateData:[],
    zoom: 12,
    center: { lat: 22.372081, lng: 114.107877 },
  };

  /**
   * Renders a shallow version of the GoogleMap Component
   */
  function renderShallow() {
    wrapper = shallow(<GoogleMapRoute {...props} />).find('GoogleMap');
  }

  it('GoogleMap Component render', () => {
    renderShallow();
    expect(wrapper).toMatchSnapshot();
  });

  it('google map without marker', () => {
    renderShallow();
    const marker = wrapper.find('Marker').length;
    expect(marker).toBe(0);
  })
});
