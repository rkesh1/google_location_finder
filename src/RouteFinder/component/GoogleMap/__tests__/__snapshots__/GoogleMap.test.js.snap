// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleMap Component Test Case  GoogleMap Component render 1`] = `ShallowWrapper {}`;
